src/esw/bno055_serial/bno055_serial.ino

```diff
@@ -0,0 +1,127 @@
+#define USE_USBCON
```

Comment:
Do we want to keep ino files in this repo? Just wondering where we draw the line between drivers and actual code to be running on the Jetson.

Reply:
I mean they aren't really getting in the way, and idk where else we would put them

Reply:
thanks, just put it into a branch called official_imu

Reply:
lol why do I have to put official in front of it

Reply:
nvm, its done

Reply:
Good question... its because if we don't write official I might accidentally delete them when I mass clean up the repo LOL